packages/amplify-e2e-tests/src/__tests__/analytics.test.ts

```diff
@@ -12,13 +12,15 @@ describe('amplify add analytics', () => {
 
   afterEach(async () => {
     await removeAnalytics(projRoot, {});
-    deleteProjectDir(projRoot);
+    await deleteProject(projRoot);
+    deleteProjectDir(projRoot), true;
```

Comment:
Why do we need this?

Reply:
Sorry I think I missed that will fix it now

Reply:
Nvm It should be there to clean the folders. I thought I had that in twice

Reply:
the deleteProjectDir() function is async?
deleteProjectDir(projRoot), true ; ? (What's the true for?)

Reply:
DeleteProjectDir cleans up the folders which is not async. DeleteProject cleans up the resources on the cloud. The true is to delete the deployment bucket

Reply:
true is outside the function call :|